gemfiles/activemodel-8.0.gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+gem "activemodel", "~> 8.0.0"
+gem "active_attr", github: "leoarnold/active_attr", branch: "leoarnold/rails-8"
```

Comment:
I'm inclined to merge this PR even though the test suite relies on your private branch at the moment. Would you mind coming back to update this after your active_attr PR is merged?

Reply:
I marked this PR as a **draft** specifically to prevent premature merging. The reason I put up the draft is so that other developers can see that this effort is already in progress and they don't need to investigate themselves (or open issues).

Reply:
Why would a _"premature"_ merge be a problem? This gem doesn't have a required runtime dependency on active_attr.  ActiveAttr is an optional dependency. 

We should be able to merge this PR and when active_attr is updated, we'll get it's benefits for free.

Reply:
i.e. Keeping this PR as a draft just holds back those projects that depend on strip_attributes but don't have a dependency on active_attr.